app/assets/stylesheets/home.scss

```diff
@@ -1,4 +1,12 @@
 // Home page New UI
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  padding-top: 90px;
+}
```

Comment:
Move these to layput.scss since these two styles will be applicable to every  pages